src/app/app-component.jsx

```diff
@@ -0,0 +1,51 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var StoreMixin = require('fluxible').StoreMixin;
+var NavigateStore = require('./nav/navigate-store');
+var navigateAction = require('./nav/navigate-action');
+var pageRegistry = require('./page-registry');
+
+/**
+ * Top-level component above all specific pages.
+ */
+
+module.exports = React.createClass({
```

Comment:
I think I like the other pattern, where you define all things locally and export the names you care about at the bottom, better.